pandas/core/indexing.py

```diff
@@ -1408,7 +1408,7 @@ def __getitem__(self, key):
             maybe_callable = com.apply_if_callable(key, self.obj)
             return self._getitem_axis(maybe_callable, axis=axis)
 
-    def _is_scalar_access(self, key):
+    def _is_scalar_access(self, key: tuple):
```

Comment:
Can you use `Tuple` from typing module and annotate types contained within if possible?

Reply:
will do